mail-client/clawsker/clawsker-1.3.1.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Applet to edit Claws Mail's hidden preferences"
+HOMEPAGE="http://www.claws-mail.org/clawsker.php"
+SRC_URI="http://www.claws-mail.org/tools/${P}.tar.xz"
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	>=dev-lang/perl-5.10
+	>=dev-perl/Gtk3-0.30
+	>=dev-perl/Locale-gettext-1.05
```

Comment:
These are always satisfied in ::gentoo tree, so the version requirements can be lifted off.